src/common/components/paymentMethods/coverFees/coverFees.tpl.html

```diff
@@ -1,4 +1,20 @@
 <div class="checkbox checkbox-inline">
+  <label ng-if="$ctrl.brandedCheckoutItem">
+    <input type="checkbox"
+           name="coverFees"
+           ng-model="$ctrl.brandedCheckoutItem.coverFees"
+           ng-value="true"
+           ng-change="$ctrl.updatePrices()">
+    <span class="cover-fees-text">
+      I want my gift to go even further for ministry!
+      To help cover credit/debit card processing fees, please adjust my gift amount to
+      {{$ctrl.brandedCheckoutItem.amountWithFee | currency}}
+      <span class="text-lowercase"
+            ng-if="$ctrl.brandedCheckoutItem.frequency && $ctrl.brandedCheckoutItem.frequency.toLowerCase() !== 'single'">
+        {{$ctrl.brandedCheckoutItem.frequency}}.
+      </span>
+    </span>
+  </label>
   <label ng-if="$ctrl.cartData.items.length === 1">
```

Comment:
Can you share this HTML? Maybe in the component, you can decide which set of data to use and set a `$ctrl.item` variable or something.

Reply:
So, if I share the HTML; the `ng-model` doesn't do what it needs to do. Do you know off-hand a way to keep that `coverFees` attribute in sync with both objects without doing it manually?

Reply:
I figured something out. It seems that because the `coverFees` is at the cart level not the item level, I need to update it there in `updatePrices()`. But, on `brandedCheckoutItem` I don't need to because it is at the item level.

Reply:
If you use the parent object, it should be passed by reference and update. Unless I'm understanding it wrong or AngularJS is doing something weird. Something like:
```ts
$onInit () {
  this.objectContainingCoverFees = this.brandedCheckoutItem || this.cartData;
}
```
and then
```ts
ng-model="$ctrl.objectContainingCoverFees.coverFees"
```

You could maybe use `ng-change` and `ng-value` instead of `ng-model`. Or bind it to a temporary variable you initialize and then update in `updatePrices`.

That's weird the data is stored inconsistently. Is that some API thing or have you been putting it in 2 different places?

Reply:
It's because branded checkout doesn't have a cart. It is just a single item. For regular checkout, the `coverFees` decision covers all gifts in the cart.

Regular checkout:
```
cart ->
  coverFees: boolean,
  items: [
    { amount, frequency, amountWithFee, etc }
  ]
```

Branded Checkout:
```
item: {
  coverFees: boolean,
  amount, frequency, amountWithFee, etc
}
```